rest-service/manager_rest/resources.py

```diff
@@ -252,8 +237,7 @@ def _process_plugins(self, file_server_root, blueprint_id):
                                         'plugins', final_zip_name)
             self._zip_dir(plugin_dir, target_zip_path)
 
-    @staticmethod
-    def _zip_dir(dir_to_zip, target_zip_path):
+    def _zip_dir(self, dir_to_zip, target_zip_path):
```

Comment:
It seems like unnecessary change.
I will change all methods that does not require access to object instance to classmethods  - they are calling each other frequently and it is easier to use cls.method_name than UploadedBlueprintsManager.method_name